.gitignore

```diff
@@ -63,5 +63,6 @@ typings/
 
 # IDE specific settings
 .idea/
+.vscode/
```

Comment:
Let's remove this.
Even if we don't want an IDE specific config it shouldn't be a part of this PR

Reply:
done